modules/manage/pages/iceberg/redpanda-topics-iceberg-snowflake-catalog.adoc

```diff
@@ -44,7 +43,7 @@ Grant privileges to the principal created in the previous step:
 . In Open Catalog, select *Catalogs*, and select your catalog.
 . On the *Roles* tab of your catalog, click *+ Catalog Role*.
 . Give the catalog role a name. 
-. Under *Privileges*, select `CATALOG_MANAGE_CONTENT`. This provides full management https://other-docs.snowflake.com/en/opencatalog/access-control#catalog-privileges[privileges] for the catalog. Then, click *Create*. 
+. Under *Privileges*, select `CATALOG_MANAGE_CONTENT`. This provides full management https://other-docs.snowflake.com/en/opencatalog/access-control#catalog-privileges[privileges^] for the catalog. Then, click *Create*. 
```

Comment:
Below: I'd change to: "To configure your Redpanda cluster to enable Iceberg on a topic and integrate with Open Catalog:"

Reply:
Updated